services/worker-manager/schemas/v1/ruleset.yml

```diff
@@ -0,0 +1,24 @@
+$schema: http://json-schema.org/draft-06/schema#
```

Comment:
This will need to point to the metaschema (see other `schemas/v1/*.yml` for an example)